en/docs/setup/changing-datasource-consent-management.md

```diff
@@ -0,0 +1,248 @@
+# Changing the Default Datasource for Consent Management
+
+WSO2 Identity Server is shipped with an embedded H2 database for storing data. 
+These default databases are located in the `<IS_HOME>/repository/database` directory 
+of the product pack.
+
+By default Consent Management data is stored in the identity database (IDENTITY_DB) with 
+Identity and UMA data. However you can change this to any database type that is 
+supported by WSO2 Identity Server.
+
+!!! note
+    For more information about databases, see 
+    [Working with Databases](../../setup/working-with-databases) and 
+    [Setting Up Separate Databases for Clustering](../../setup/setting-up-separate-databases-for-clustering).
+
+Following are the sample configuration for each database type.
+
```

Comment:
Once when we separate out consent database from WSO2IdentityDB, we need to specify it in 
wso2is-5.9.0/repository/conf/consent-mgt-config.xml

--------
[authentication.consent]
data_source = "WSO2CONSENT_DB"
---
Add the toml config for that as well in each example

Reply:
Fixed : 380b00c31c88d7a4de4a940ca60310a3367e5c37